typeql-src/modules/ROOT/pages/schema/define-types.adoc

```diff
@@ -273,15 +281,20 @@ To add roles that entities of a specific entity type can play, use the `plays` k
 ----
 define
 
-access sub relation, relates object;
-object sub entity, abstract, plays access:object;
+access sub relation,
+    relates object;
+
+object sub entity,
+    abstract,
+    plays access:object;
 ----
 
 ==== Subtypes another entity
 
 [NOTE]
 ====
-All types that are subtyping `entity` built-in type directly or through other subtypes are called entity types. Instances of these types are called entities. The same approach can be applied to attributes and relations.
+All types that are subtyping `entity` built-in type directly or through other subtypes are called entity types.
+Instances of these types are called entities. The same approach can be applied to attributes and relations.
```

Comment:
Should "Approach" be "terminology"? -- eg "Attributes and relations use analogous terminology". Or something?

Is it "can" or "is/are/should be" (more assertive?)

Reply:
```suggestion
Instances of entity types are called entities. A similar terminology is applied to attributes and relations.
```